helm-charts/agentqna/Chart.yaml

```diff
@@ -7,37 +7,37 @@ description: The Helm chart to deploy AgentQnA
 type: application
 dependencies:
   - name: agent
-    version: 1.0.0
+    version: 0-latest
```

Comment:
Why `0-latest` instead of `1.1.0`?

Reply:
Using 0 as version number seems rather misleading.  I think something like `1.2-dev`, `1.2-rc` or `1.1.0-latest` would be better.

Reply:
Best would be if "NeuroBot" would do such update to `main` automatically, right after it has updated versions for the release branch...

Reply:
There should be just one development version in the packages (https://github.com/orgs/opea-project/packages/container/package/charts%2Fchatqna). If we use 1.2-dev, there would be more versions like 1.3-dev remains, and it's hard to cleanup. Actually 1.2-dev was the name I first used, but think 0-latest from @poussa is a better one. I'm open to other good proposals.

There should be no such update on main branch after this one merged, so NeuroBot would not be necessary.

Reply:
> Actually 1.2-dev was the name I first used, but think 0-latest from @poussa is a better one.

@poussa Why `0-latest` instead of `latest`?